app/views/public_pages/home.html.erb

```diff
@@ -36,7 +41,11 @@
         </h1>
         <p>
           <strong>
-            <%= open_for_intake? ? t("views.public_pages.home.subheader") : t("views.public_pages.home.subheader_offseason") %>
+            <% if open_for_intake? %>
```

Comment:
it might be helpful to rename `open_for_intake?` to `open_for_gyr_intake?` so that this is less confusing